packages/clicker-ui/src/App.js

```diff
@@ -64,31 +67,32 @@ const Clicker = ({ name }) => {
     })
   }, [name])
 
+  // Updates Action Stack
   useInterval(async () => {
     if (game.state.actions.length) {
       const synced = await syncGame(game)
       setGame(game.fastForward(synced))
     }
   }, 3 * 1000)
 
-  const clickHandler = async () => {
-    game.click()
+  // Increments score based on passive income, once per second
+  useInterval(async () => {
```

Comment:
Instead of a mutating the game state every time, call `getCurrentState` on an interval and use that in the render.